deploy/alerts/simple-comparison.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: stf-default
+    role: alert-rules
+  name: prometheus-alarms-part-two
+  namespace: service-telemetry
+spec:
+  groups:
+    - name: ./openstack.rules
+      rules:
+        - alert: high cpu
+          labels:
+            severity: warn
+          annotations:
+            summary: CPU usage high (warning)
+          expr: >-
+            sum without(cpu,type) (collectd_cpu_percent{type=~"user|system"}) / sum without(cpu,type) (collectd_cpu_percent{type="idle"}) > 0.5
+          for: 10m
+        - alert: high cpu
+          labels:
+            severity: critical
+          annotations:
+            summary: CPU usage high (critical)
+          expr: >-
+            sum without(cpu,type) (collectd_cpu_percent{type=~"user|system"}) / sum without(cpu,type) (collectd_cpu_percent{type="idle"}) > 0.7
+          for: 10m
+        - alert: inode usage
+          labels:
+            severity: warning
+          annotations:
+            summary: Inodes usage (warning)
+          expr: >-
+            sum without (endpoint,service,type) (collectd_df_df_inodes{df="root",type="used"})/ (sum without (endpoint,service,type) (collectd_df_df_inodes{df="root",type=~"free|used"})) > 0.6
+          for: 10m
+        - alert: inode usage
+          labels:
+            severity: critical
+          annotations:
+            summary: Inodes usage (critical)
+          expr: >-
+            sum without (endpoint,service,type) (collectd_df_df_inodes{df="root",type="used"})/ (sum without (endpoint,service,type) (collectd_df_df_inodes{df="root",type=~"free|used"})) > 0.6
```

Comment:
The critical alarm should have a higher threshold than the warning. Suggest 0.8 maybe?

Reply:
Oh yes, good catch. That is a typo and should be 0.8, indeed.